src/metabase/pulse/render/png.clj

```diff
@@ -23,6 +23,21 @@
            org.fit.cssbox.layout.BrowserCanvas
            org.w3c.dom.Document))
 
+
+(defonce
+  ^{:doc     "Makes custom fonts available to Java so that CSSBox can render them"
+    :private true}
+  register-fonts
+  (delay (doseq [weight ["regular" "700" "900"]]
```

Comment:
You just _know_ that, if the set of fonts ever change, that this vector won't be updated.

Isn't it better, then, to do something like this?

```clojure
(->> (clojure.java.io/file "resources/frontend_client/app/fonts/lato-v16-latin")
     file-seq
     (filter #(re-find #"^lato-v16-latin-\w+\.ttf$" (.getName %))))
```

But this suffers from you-can't-list-files-in-a-jar's-resources syndrome, right?

Reply:
yeah, the code won't pick up new fonts :( however i think for now i'd like to go ahead and merge the PR anyway. we could investigate some scheme for discovering and loading fonts, but i don't think that's a high priority